src/Enzyme.jl

```diff
@@ -176,6 +178,7 @@ Enzyme.autodiff(ReverseWithPrimal, x->x*x, Active(3.0))
     point values, but cannot do so for integer values in tuples and structs.
 """
 @inline function autodiff(::ReverseMode{ReturnPrimal, RABI}, f::FA, ::Type{A}, args...) where {FA<:Annotation, A<:Annotation, ReturnPrimal, RABI<:ABI}
+    JuliaContext() do ctx
```

Comment:
This should probably be in the generated function code, not here.

Reply:
It looks like there are multiples calls to these in autodiff* top-level functions and I end up having to spin up some contexts to allow the returned thunks to be called. This way also results in fewer calls to activate/deactivate contexts that reach into task-local storage.